Allow standard to be selected in composer.json (#1)
php-composter/php-composter-phpcs-wpcs: README.md

## Internals

WPCS provides 5 different coding standards (`WordPress-VIP`, `WordPress`,
`WordPress-Extra`, `WordPress-Docs` and `WordPress-Core`). By default this uses
`WordPress-Extra`. If you want to use another one of the standards, you can

> specify it in your project's composer.json's `extra` key:

Reviewer: Reorder for readability
```suggestion
specify it in the `extra` key of your project's `composer.json` file:
```
